Makefile

```diff
@@ -68,9 +68,9 @@ build/operator-sdk build/ansible-operator build/helm-operator:
 
 ##@ Dev images
 
-# Convenience wrappers for building/pushing all images.
+# Convenience wrapper for building all remotely hosted images.
 .PHONY: image-build
-IMAGE_TARGET_LIST := operator-sdk helm-operator ansible-operator scorecard-test scorecard-test-kuttl custom-scorecard-tests
```

Comment:
Is not the go-e2e-tests using it?

Reply:
Yes, it is built explicitly with `build/custom-scorecard-tests`